search/search_tests_support/helpers.cpp

```diff
@@ -64,6 +64,13 @@ bool SearchTest::ResultMatches(search::Result const & result, Rule const & rule)
   return tests_support::ResultMatches(m_dataSource, rule, result);
 }
 
+bool SearchTest::ResultsNumber(string const & query, string const & locale, size_t number)
```

Comment:
s/ResultsNumber/NumberOfResults/

Reply:
or return the number and check it in the call place

Reply:
GetResultsNumber